snugget_load.py

```diff
@@ -226,8 +225,8 @@ def findAllFilterVals(shapefile):
   return shapefile.objects.values_list(fieldName, flat=True)
 
 
-def checkForSnugget(shapefile, section, filterColumn, filterVal):
-  kwargs = {'section': section, filterColumn: getShapefileFilter(shapefile, filterVal)}
+def checkForSnugget(shapefile, section, order, filterColumn, filterVal):
```

Comment:
I think that adding `order` to the fields that we use when checking for an existing snugget may have an undesirable effect: if we try to change the order of some snuggets, won't this mean that they get duplicated between the old order and the new, because this function doesn't see the old and new as the same snugget?

Reply:
Ooh, you're right.

I guess instead of making order part of how we uniquely identify snuggets, we can add some options to the part where we ask the user about replacement:

- replace the old one with this one, which has a different order
- add this one because it's a different snugget

Is that the least confusing way to do things? Also, we probably shouldn't ask at all if the only one present is the one we just loaded from this file, but it's hard to tell the difference between snuggets that have existed for a long time, and ones that have just existed because they were loaded during this process.

Reply:
Oh, right.  This adds more complexity overall than I had understood.  I hadn't appreciated that adding the `order` attribute creates a situation in which we could legitimately have multiple snuggets for the same section and value in one file.  It seems worth us discussing in person on Monday - does waiting for that hold you up excessively?

Reply:
No, i don't think so- i can start styling work without this in.

Reply:
We resolved this with an offline conversation - I now think it makes sense to do this the way it is here.